beacon-chain/blockchain/service.go

```diff
@@ -37,3 +45,28 @@ func (c *ChainService) Stop() error {
 	log.Info("Stopping blockchain service")
 	return nil
 }
+
+// updateActiveState receives a beacon block, computes a new active state and writes it to db
+func (c *ChainService) updateActiveState() {
+	for {
+		select {
+		case block := <-c.beaconBlock:
+			log.Debugf("Received beacon block, last active state hash: %v", block.ActiveStateHash)
```

Comment:
Use the new structure of `log.WithFields(logrus.Fields{"activeStateHash": block.ActiveStateHash}).Debug("Received beacon block")`